images/linux/Ubuntu2004-Readme.md

```diff
@@ -92,44 +91,44 @@ to accomplish this.
 - HHVM (HipHop VM) 4.172.1
 - jq 1.6
 - Kind 0.18.0
-- Kubectl 1.26.3
+- Kubectl 1.27.1
 - Kustomize 5.0.1
 - Leiningen 2.10.0
 - MediaInfo 19.09
 - Mercurial 5.3.1
 - Minikube 1.30.1
-- n 9.0.1
+- n 9.1.0
 - Newman 5.3.2
 - nvm 0.39.3
 - OpenSSL 1.1.1f-1ubuntu2.17
-- Packer 1.8.6
+- Packer Packer 1.8.6
```

Comment:
Look like we need fix Get-PackerVersion function in https://github.com/actions/runner-images/blob/5657dec7af3b55531d12f753cbd04370357941de/images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1